app/models/form_item.rb

```diff
@@ -100,8 +100,10 @@ def descendant_questionings(nodes = nil)
     end
   end
 
-  def preordered_descendants
-    self.class.sort_by_ancestry(descendants.order(:rank)) { |a, b| a.rank <=> b.rank }
+  def preordered_descendants(eager_load: nil)
+    rel = descendants.order(:rank)
```

Comment:
I don't know what 'rel' stands for here? Could you comment or update for future devs?